app/src/main/res/values-ru/arrays.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+    <string-array name="themesEntries">
+        <item>Светлая (по умолчанию)</item>
+        <item>Полупрозрачная</item>
+        <item>Прозрачная</item>
+        <item>Чёрная</item>
+        <item>Чёрная полупрозрачная</item>
+        <item>Чёрная прозрачная</item>
+    </string-array>
+    <string-array name="themesValues" translatable="false">
```

Comment:
Please do not include the items that have translatable=false in your file :)

Reply:
Will be fixed! Thanks!